devel/Dockerfile

```diff
@@ -57,6 +58,23 @@ RUN if [ "$TARGETPLATFORM" != "linux/arm/v7" ]; then \
         cd ../.. && rm -rf qemu; \
     fi
 
-ENV PATH="/opt/phoenix/utils:${PATH}"
+# install Jlink tools for 32-bit armv7 architecture and 64-bit amd architecture
+RUN if [ "$TARGETPLATFORM" = "linux/arm/v7" ]; then \
+        JLINK_FILENAME="JLink_Linux_arm"; \
+    elif [ "$TARGETPLATFORM" = "linux/amd64" ]; then \
+        JLINK_FILENAME="JLink_Linux_x86_64"; \
+    fi \
+    mkdir -p /opt/SEGGER && \
+    (cd /opt/SEGGER && \
+    wget --no-check-certificate --post-data "accept_license_agreement=accepted" "https://www.segger.com/downloads/jlink/${JLINK_FILENAME}.tgz" && \
```

Comment:
This works on `amd64` platform.

I've narrowed it down to missing CA hash symlinks in `/etc/ssl/certs/`. I don't know why they are missing, but fortunately they are easy to add.

Please add `&& c_rehash   # needed for armv7 CA certs to work` as a line 24 (and `\` at the end of line 23)

Reply:
Thanks, it seems to work fine without `--no-check-certificate` now.